lta/nersc_mover.py

```diff
@@ -2,6 +2,7 @@
 """Module to implement the NerscMover component of the Long Term Archive."""
 
 import asyncio
+from logging import Logger
```

Comment:
Our integration with LGTM doesn't like that we use `from X import Y` and `import X`.
Could you remove this import and restore `Logger = logging.Logger` below?